ui/src/containers/Layout/Layout.scss

```diff
@@ -0,0 +1,8 @@
+@import 'Src/common/variables.scss';
```

Comment:
path name is case-sensitive in specific environment. probably correct this with alias I saw you used in another .scss file:
```
 @import '@src/common/variables.scss';
```

Reply:
looks like we will have 2 duplicated variable directory, `src/common/variables.scss` and `src/style/common.scss`

Reply:
oops, I'll keep the`style/common.css` 👽

Reply:
fix in https://github.com/aStanleyLiang/luckydraw/pull/14/commits/5a26b66564d015ef77ad5fcce19b6fdbb2bc745e thx~

Reply:
@aMichaelPoernomo  nice catch